django_cloud_deploy/cloudlib/service_account.py

```diff
@@ -85,6 +86,21 @@ def _generate_updated_iam_policy(self, policy, member: str, role: str):
         policy['bindings'].append(new_bindings)
         return policy
 
+    @staticmethod
+    def _not_conflict_code(code: int) -> bool:
```

Comment:
Does this work? Won't this receive a HttpError instance?

You might also need a test for this.

Reply:
Fixed this and added unittest for this function.